src/types/types.tsx

```diff
@@ -19,3 +19,10 @@ export enum VoucherType {
   ORANGE,
   PURPLE,
 }
+
+export type Transaction = {
+  date: string;
+  uuid: uuid;
+  vendorUUID: uuid;
+  vouchers: Array<uuid>;
```

Comment:
Not sure line 27 is accurate. Is the type of **vouchers** supposed to be an Array of uuid's?